flutter_local_notifications/lib/src/flutter_local_notifications_plugin.dart

```diff
@@ -191,6 +191,10 @@ class FlutterLocalNotificationsPlugin {
       return;
     }
     if (defaultTargetPlatform == TargetPlatform.android) {
+      assert(
```

Comment:
Actually in hindsight, I believe the more appropriate fix is for `AndroidNotificationDetails` to be non-nullable in the method signatures of methods for display/scheduling notifications on Android. I'll close this PR as there's a 10.0.0 pre-release i'm working on where this (breaking) change can be done